brig/cmd/brig/commands/check.go

```diff
@@ -0,0 +1,118 @@
+package commands
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	apps_v1 "k8s.io/api/apps/v1"
+	meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func init() {
+	Root.AddCommand(check)
+}
+
+const (
+	checkUsage = `Checks the status of your Brigade installation
+
+Specifically, it checks if the Controller, API Server and Kashti deployments have more than one available replicas (pods).
```

Comment:
Maybe `at least one` (or `exactly one`?) instead of `more than one`?

Reply:
oh, nice catch, editing!